packages/core/src/components/overlay/overlay.tsx

```diff
@@ -69,6 +69,13 @@ export interface IOverlayableProps extends IOverlayLifecycleProps {
      */
     usePortal?: boolean;
 
+    /**
+     * The container that the overlay renders its contents to.
```

Comment:
"The container element into which the overlay renders its contents, when `usePortal` is `true`. This prop is ignored if `usePortal` is `false`.